tests/python/unittest/test_operator.py

```diff
@@ -2506,8 +2506,9 @@ def test_stn_valid_sampling():
     ) + target_shape))
 
 
-# Seed set because the test is not robust enough to operate on random data
-@with_seed(1234)
+# @haojin2: Getting rid of fixed seed as flakiness could not be reproduced,
+# tracked at https://github.com/apache/incubator-mxnet/issues/11714
```

Comment:
Feel free to remove that comment

Reply:
Let's still keep it here, in case if it fails on some other platforms, or we detect any regression on this operator in the future. What do you think?

Reply:
Sounds good to me